src/app/vertextool/qgsvertextool.h

```diff
@@ -61,7 +61,7 @@ class APP_EXPORT QgsVertexTool : public QgsMapToolAdvancedDigitizing
 {
     Q_OBJECT
   public:
-    QgsVertexTool( QgsMapCanvas *canvas, QgsAdvancedDigitizingDockWidget *cadDock );
+    QgsVertexTool( QgsMapCanvas *canvas, QgsAdvancedDigitizingDockWidget *cadDock, bool activeLayerOnly = false );
 
```

Comment:
can you rather create an enum, it's more future proof.
```c++
enum VertexToolMode {
  ActiveLayer,
  AllLayers
};
```

Reply:
You're right, sounds good this way. I'll do that!

Reply:
done